v2/emailpassword/serverless/with-aws-lambda/api-gateway-config.mdx

```diff
@@ -1,43 +0,0 @@
----
```

Comment:
instead of getting rid of this file, remove all the content and just put a `<Redirector .. >` component here pointing to the new page. You don't need to mention this file in sidebar.json either. 

This is so that if Google links to this page anywhere, it won't show a 404 and instead will redirect the user to the right page.

Reply:
done